lib/shared/logger.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:logger/logger.dart';
+
+var logger = Logger(
+  filter: ProductionFilter(),
+  printer: SimplePrinter(colors: false),
```

Comment:
`colors` are disable here because they don't play nicely with VSCode's debug output pane.

There are other [printers](https://github.com/SourceHorizon/logger?tab=readme-ov-file#output) as well like `PrettyPrinter`, but they get a bit too spammy for my taste.